hazelcast/src/main/java/com/hazelcast/monitor/impl/NearCacheStatsImpl.java

```diff
@@ -208,6 +227,7 @@ public JsonObject toJson() {
         root.add("lastPersistenceDuration", lastPersistenceDuration);
         root.add("lastPersistenceWrittenBytes", lastPersistenceWrittenBytes);
         root.add("lastPersistenceKeyCount", lastPersistenceKeyCount);
+        root.add("lastPersistenceFailure", lastPersistenceFailure == null ? NULL_MARKER : lastPersistenceFailure);
```

Comment:
It seems not supported to store `null` in a `JsonObject`, at least I got exceptions when I did this. This is the reason for this stupid `NULL_MARKER`...